Dockerfile

```diff
@@ -1,44 +1,37 @@
-# This Dockerfile allows to build the package-registry and packages together.
-# It is decoupled from the packages and the registry even though for now both are in the same repository.
+# This image contains the package-registry binary.
+# It expects packages to be mounted under /packages or have a config file loaded into /package-registry/config.yml
+
+# Build binary
 ARG GO_VERSION=1.14.2
-FROM golang:${GO_VERSION}
+FROM golang:${GO_VERSION} AS builder
 
-# Get dependencies
-RUN \
-    apt-get update \
-      && apt-get install -y --no-install-recommends zip rsync \
-      && rm -rf /var/lib/apt/lists/*
+ENV GO111MODULE=on
+COPY ./ /package-registry
+WORKDIR /package-registry
+RUN go build .
 
-# Copy the package registry
-COPY ./ /home/package-registry
-WORKDIR /home/package-registry
 
-ENV GO111MODULE=on
-RUN go get -u github.com/magefile/mage
-# Prepare all the packages to be built
-RUN mage build
+# Run binary
+FROM centos:7
 
-# Build binary
-RUN go build .
+# Get dependencies
+RUN yum install -y zip rsync && yum clean all
 
-# Move all files need to run to its own directory
-# This will become useful for staged builds later on
-RUN mkdir -p /registry/public # left for legacy purposes
-RUN mkdir -p /registry/packages/package-storage
-RUN mv package-registry /registry/
-RUN cp -r build/package-storage/packages/* /registry/packages/package-storage/
-RUN cp config.docker.yml /registry/config.yml
+# Move binary from the builder image
+COPY --from=builder /package-registry/package-registry /package-registry/package-registry
 
 # Change to new working directory
-WORKDIR /registry
+WORKDIR /package-registry
```

Comment:
Keep in mind that changing this path may break depending docker-compose definitions:

https://github.com/elastic/integrations/blob/master/testing/environments/snapshot.yml#L46
https://github.com/elastic/integrations/blob/master/testing/environments/kubernetes/snapshot.yml#L193

Reply:
Got it. Assuming we get https://github.com/elastic/integrations/pull/156 in first, we could roll it out in two steps? I will update https://github.com/elastic/integrations/pull/156 to also adjust the k8s one to a fix registry.